db/seeds.d/100-installation_media.rb

```diff
@@ -12,7 +12,8 @@
     { :name => "FreeBSD mirror",       :os_family => "Freebsd", :path => "http://ftp.freebsd.org/pub/FreeBSD/releases/$arch/$version-RELEASE/" },
     { :name => "OpenSUSE mirror",      :os_family => "Suse",    :path => "http://download.opensuse.org/distribution/leap/$version/repo/oss", :operatingsystems => os_suse },
     { :name => "Ubuntu mirror",        :os_family => "Debian",  :path => "http://archive.ubuntu.com/ubuntu" },
-    { :name => "CoreOS mirror",        :os_family => "Coreos",  :path => "http://$release.release.core-os.net" }
+    { :name => "CoreOS mirror",        :os_family => "Coreos",  :path => "http://$release.release.core-os.net" },
+    { :name => "RancherOS mirror", :os_family => "Rancheros", :path => "https://github.com/rancher/os/releases/download/v$version" }
```

Comment:
This line is a bit off, but I will merge anyway, tests are green (and costly).

Out of curiousity, this is the very first HTTPS media, I assume you used some local mirror while you were testing this? Because this won't work by default, unless you have github CA certificate installed into OS certificate bundle so wget won't refuse to connect when downloading PXE bits.

Reply:
And I take it back, we do have `--no-check-certificate` for wget there.

Reply:
Yes ! Https works because you already anticipate that :)